### DISCUSSION TITLE
[译]ProseMirror 的由来

### DISCUSSION THREAD

**Discussion by Xheldon** [Xheldon/x_blog#115] [Announcements]
https://www.xheldon.com/tech/ProseMirror.html 

 原文作者写于 2015 年 7 月 7 日，作者决定要写一个富文本框架的时候写的一片文章，两年后 ProseMirror 1.0 发布，本文偏意译。

**Comment by Benbinbin** [comment]
感谢博主的用心翻译，这里分享一个视频 https://youtu.be/EEF2DlOUkag 是 Marijn Haverbeke 在 2015 年的 JSConf 大会上介绍 ProseMirror 的演讲

**Comment by Xheldon** [comment]
> @Benbinbin
> 感谢博主的用心翻译，这里分享一个视频 https://youtu.be/EEF2DlOUkag 是 Marijn Haverbeke 在 2015 年的 JSConf 大会上介绍 ProseMirror 的演讲

这个视频看过，那个时候 Marijn 还很腼腆的样子🤣
